src/sentry/static/sentry/app/views/settings/project/projectFilters/projectFiltersSettings.jsx

```diff
@@ -171,22 +179,47 @@ class ProjectFiltersSettings extends AsyncComponent {
     onBlur(subfilters, e);
   };
 
-  renderDisabledFeature(fields) {
-    let {project, organization} = this.props;
-    return this.state.hooksDisabled.map(hook =>
-      hook(organization, project, null, fields)
-    );
-  }
-
   renderBody() {
-    let {features, params} = this.props;
+    let {organization, features, params} = this.props;
     let {orgId, projectId} = params;
     let {project} = this.state;
 
     if (!project) return null;
     let projectEndpoint = `/projects/${orgId}/${projectId}/`;
     let filtersEndpoint = `${projectEndpoint}filters/`;
 
+    let disabledAdvFilters = p => (
+      <FeatureDisabled
+        feature={p.features[0]}
+        alert={PanelAlert}
+        name={t('Custom Inbound Filters')}
+        message={t(
+          'Release and Error Message filtering are not enabled on your Sentry installation'
+        )}
+      />
+    );
+
+    let advCustomFilters = (
```

Comment:
Let's make this and `disableAdvFilters` into methods on the component class. Also can we spell out Advanced? (Do we even need to call it "advanced"?)